app/models/effective/datatables/logs.rb

```diff
@@ -37,11 +55,17 @@ class Logs < Effective::Datatable
         # A nil attributes[:log_id] means give me all the top level log entries
         # If we set a log_id then it's for sub logs
         def collection
-          if attributes[:user_id].present?
-            Effective::Log.unscoped.where(:parent_id => attributes[:log_id]).where(:user_id => attributes[:user_id]).includes(:user)
-          else
-            Effective::Log.unscoped.where(:parent_id => attributes[:log_id]).includes(:user)
-          end
+          col = Effective::Log.unscoped
+            .where(parent_id: attributes[:log_id])
+            .joins('LEFT JOIN users ON users.id = logs.user_id')
```

Comment:
Why did you change `.inlcudes(:user)` to `.joins('LEFT JOIN users ON users.id = logs.user_id')`?  I would much prefer we stick to the Rails conventions rather than using custom SQL joins.  Also, it depends how smart Rails is about when it performs the SQL calls but it looks like the new method will perform two SQL queries if `attributes[:user_id].present?` is `true`.  I prefer the original code to this refactor if two SQL queries get performed.